Configs/.config/dunst/dunstrc

```diff
@@ -309,6 +329,35 @@
     # where there are multiple screens with very different dpi values.
     per_monitor_dpi = false
 
+
+[urgency_low]
+    # IMPORTANT: colors have to be defined in quotation marks.
+    # Otherwise the "#" and following would be interpreted as a comment.
+    background = "#6591A380"
+    foreground = "#CCF0FFE6"
+    frame_color = "#6591A303"
+    timeout = 5
+    # Icon for notifications with low urgency, uncomment to enable
+    default_icon = "$HOME/.config/dunst/icons/hyprdots.svg"
+
+[urgency_normal]
+    background = "#3A4A6B80"
+    foreground = "#CCDDFFE6"
+    frame_color = "#3A4A6B03"
+    timeout = 5
+    #override_pause_level = 30
+    # Icon for notifications with normal urgency, uncomment to enable
+    default_icon = "$HOME/.config/dunst/icons/hyprdots.svg"
```

Comment:
I realize we're still using hyprdots icons for dunst notifications.

I don't have the svg file to replace this, @prasanthrangan just going to let you know.